src/index.ts

```diff
@@ -167,11 +172,9 @@ export default class OtaClient {
     async getTranslations(): Promise<Translations> {
         const languages = await this.listLanguages();
         const translations: Translations = {};
-        await Promise.all(
-            languages.map(async language => {
-                translations[language] = await this.getLanguageTranslations(language);
-            }),
-        );
+        for (const language of languages) {
```

Comment:
How does this improves the performance? In previous example `getLanguageTranslations` was executed per each language in parallel (independently). And now here each  execution of `getLanguageTranslations` blocks next iteration.

Reply:
Yeah I forgot to test this and turns out for loops don't all run at once like I thought they did. The usage of Promise.all still seems weird since we're working with an object and not an array but I'll revert these for now

Reply:
FYI `Promise.all` is used here exactly to achive this concurrency.  We are iterating over languages array and executing async operation and with usage of `Array.map` function gives us an array of promises and in order to wait for them all we are using `Promise.all` here.

Reply:
Yeah I know but the use of await and Promise.all simultaneously usually defeats its purpose. Guess this is still somewhat fast so it’s fine

Reply:
Not sure on which documentation you are relying but here is a nice example of `Promise.all` with `async/await` https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await#awaiting_a_promise.all and you will not achive the same without `Promise.all`. And here in library we have the same situation.